BN re-export failure

Hello! I'm not being able to use the reexported version of BN.js from TypeScript due to [how the export is defined](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/externals.ts#L7). I'm using the `7.0.3` release.

Example code:
```typescript
import { BN } from 'ethereumjs-util';

function hexStringIsZero(hexNumber: string): boolean {
  return new BN(hexNumber, 'hex').isZero();
}
```

Printed error plus suggested fix:
```
../core/src/Foo.ts:51:14 - error TS2351: This expression is not constructable.
  Type 'typeof BN' has no construct signatures.

51   return new BN(hexNumber, 'hex').isZero();
                ~~

  ../core/node_modules/ethereumjs-util/dist/externals.d.ts:6:1
    6 import * as BN from 'bn.js';
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Type originates at this import. A namespace-style import cannot be called or constructed, and will cause a failure at runtime. Consider using a default import or import require here instead.
```

--- a/src/externals.ts
+++ b/src/externals.ts
@@ -7,8 +7,8 @@
 // TODO: This can be replaced with a normal ESM import once
 // the new major version of the typescript config package
 // is released and adopted here.
-import BN = require('bn.js');
-import rlp = require('rlp');
+import BN = require('bn.js')
+import rlp = require('rlp')
 
 /**
  * [`BN`](https://github.com/indutny/bn.js)

--- a/src/externals.ts
+++ b/src/externals.ts
@@ -4,8 +4,11 @@
  * @packageDocumentation
  */
 
-import * as BN from 'bn.js'
-import * as rlp from 'rlp'
+// TODO: This can be replaced with a normal ESM import once
+// the new major version of the typescript config package
+// is released and adopted here.
+import BN = require('bn.js');
+import rlp = require('rlp');
 
 /**
  * [`BN`](https://github.com/indutny/bn.js)

